plugins/pcap/CHANGELOG.md

```diff
@@ -2,6 +2,23 @@
 
 This changelog documents all notable changes to the PCAP plugin for VAST.
 
+## v2.0.0
+
+### :gift: Features
+
+- VAST no longer discards parsed 802.1Q VLAN tags but instead writes them into
+  a new record `vlan`. This record contains two fields `outer` and `inner` that
+  correspond to the outer and inner VLAN identifier (VID), respectively.
+  [#2179](https://github.com/tenzir/vast/pull/2179)
+
+### :zap: Breaking Changes
```

Comment:
I think this is just a change, not a breaking change.

```suggestion
### :warning: Changes
```

Reply:
It's breaking in that the packet layout is now different: the `community_field` is there even when you add `--disable-community-id`. The flag only change the value from the string to `null`.

Reply:
That's not a breaking change, no? Adding a field should be backwards compatible in the type registry.